core/src/test/java/apoc/coll/CollTest.java

```diff
@@ -40,6 +41,22 @@
 import static org.junit.Assert.*;
 
 public class CollTest {
+    // query that procedures a list,
+    // with both entity types, via collect(..), and hardcoded one
+    private static final String QUERY_WITH_MIXED_TYPES = """
+            WITH COLLECT {
+            MATCH (n:Test)
+            RETURN {something: n.something}
+            ORDER BY n.a} + { something: [] } + {something: 'alpha'} + {something: [1,2,3]} AS collection\s
```

Comment:
What is the \s here for?

Reply:
It was just a whitespace which intellij had automatically added. 
Removed :)